src/vsg/io/VSG.cpp

```diff
@@ -99,7 +99,7 @@ vsg::ref_ptr<vsg::Object> VSG::read(const vsg::Path& filename, ref_ptr<const Opt
         vsg::Path filenameToUse = findFile(filename, options);
         if (!filenameToUse) return {};
 
-        std::ifstream fin(filenameToUse, std::ios::in | std::ios::binary);
+        std::ifstream fin(filenameToUse.c_str(), std::ios::in | std::ios::binary);
```

Comment:
I presume the Ming std::ifstream must be missing a (wstring&) constructor. Would adding a vsg::Path operator:

        inline operator const value_type* () const noexcept { return _string.c_str(); }

work?

I think this might be preferable to updating lots of VSG and associated project code  for just MinGW.